下载时出现错误怎么办

remote: Permission to lily-abaixx/github-actions-youtube-dl.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/lily-abaixx/github-actions-youtube-dl/': The requested URL returned error: 403
Error: Process completed with exit code 128.


不知道作者还在吗？请问这种报错怎么办。（是电脑小白，如果问了很傻的问题，我很抱歉。。）

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -5,6 +5,10 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      
     steps:
       - name: Clone repository
         uses: actions/checkout@v3

